### ISSUE TITLE
dolos bootstrap - progress for creating indexes

### ISSUE THREAD

**Issue by MadOrkestra** [txpipe/dolos#322]
Most likely expected behaviour that index creation takes quite a bit of time, but would be nice to have some kind of progress shown, so you know if dolos crashed or if it is still doing something (I suspect it is, top shows dolos at 98% CPU usage):

<img width="995" alt="Screenshot 2024-08-07 at 21 07 27" src="https://github.com/user-attachments/assets/4f680713-cf38-4820-a8bc-efc8e5d4c49c">

**Comment by MadOrkestra** [comment]
lol, successfully finished two minutes after I hit send. 1hr 9 minutes for index creation. 

Still would be nice if this could show progress.

**Comment by scarmuega** [comment]
@MadOrkestra yes, agreed. I'm adding to the backlog but in the short-term I'll providing a warning in the docs.

**Comment by smoothyetimountainclub** [comment]
Posted this in the Discord, but would be great to add a message after the snapshot is done downloading as well.

![image](https://github.com/user-attachments/assets/3d2038af-2fc4-4abb-9daf-bda61735820e)

After the snapshot is done downloading.. it just hangs here for a few minutes. It should show a log saying it's doing something.. so it doesn't look like it froze!

**Comment by scarmuega** [comment]
Snapshot/bootstrap progress reporting has since been improved (e.g. the live snapshot progress bar). Closing as likely addressed — please reopen if the index-creation phase still hangs without any progress indication on a current release.
